lua/ttt2/libraries/none.lua

```diff
@@ -86,7 +86,7 @@ end
 -- @return boolean
 -- @realm shared
 function IsPlayer(ent)
-	return ent and IsValid(ent) and ent:IsPlayer()
+	return ent and isentity(ent) and IsValid(ent) and ent:IsPlayer()
```

Comment:
I suppose this solves some kind of edge-case? Could you explain what that edge-case is?

Also this would be a change in behaviour and should be noted in the changelog.

Reply:
IsPlayer checks if a entity is a player. However if you give a string or a number to that function, it errors. So I added this check to simplify the code

Reply:
tbh that qualifies as a bug fix then :D (and should be in the changelog)